File: QB_Challenges/PrepInsta/CPP/04_Sum_Between_Range/SORN_BruteForce.cpp
#include <iostream>
using namespace std;


void SORN(){
    int num1, num2;
    cout <<"Enter the first number: ";
    cin >> num1;
    cout <<"Enter the second number: ";
    cin >> num2;

    int sum;
    for(int i=num1; i<=num2; i++){
        sum += i;
    }
    cout <<"The sum of "<<num1<<" & "<<num2<<" is: "<<sum<<endl;
}

int main(){
    SORN();

    return 0;
}

File: QB_Challenges/PrepInsta/01_Getting_Started/04_Sum_Between_Range/Java/SORN_BruteForce.java
import java.util.Scanner;

public class SORN_BruteForce{

    public static void sorn() {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the first number: ");
        int num1 = sc.nextInt();
        System.out.print("Enter the second number: ");
        int num2 = sc.nextInt();
        sc.close();

        int sum = 0;
        for(int i = num1; i<=num2; i++){
            sum += i;
        }

        System.out.println("The sum of "+num1+" & "+num2+" is: "+sum);
    
    }

    public static void main(String[] args) {
        sorn();
    }
}